odom_tf_node:
  ros__parameters:
    odom_topic: /odom
    parent_frame: odom
    child_frame: base_footprint